libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -6455,6 +6455,25 @@ void ccPointCloud::clearLOD()
 	}
 }
 
+bool ccPointCloud::hasUsableLOD() const
+{
+	if(m_lod && m_lod->isInitialized())
```

Comment:
```suggestion
	if (m_lod && m_lod->isInitialized())
```

Reply:
And it will be the same everywhere, to be consistent with CC's code ;)

Reply:
very sorry. The clang format file definition would definitely help. I'm working on multiple project and language and it's hard to make the switch every time...